src/ucp/core/ucp_types.h

```diff
@@ -30,6 +30,7 @@ UCP_UINT_TYPE(UCP_MD_INDEX_BITS)     ucp_md_map_t;
 
 /* Lanes */
 #define UCP_MAX_LANES                8
+#define UCP_MAX_RAILS                4
```

Comment:
why 4?

Reply:
actually 2 lanes is enough, but we decided to multiply this value to 2x for future usage